src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -589,9 +589,11 @@ export function isEventFromBrowserJavaScriptSDK(event: SentryTransactionEvent):
     return false;
   }
   // based on https://github.com/getsentry/sentry-javascript/blob/master/packages/browser/src/version.ts
-  return ['sentry.javascript.browser', 'sentry.javascript.react'].includes(
-    sdkName.toLowerCase()
-  );
+  return [
+    'sentry.javascript.browser',
+    'sentry.javascript.react',
+    'sentry.javascript.gatsby',
+  ].includes(sdkName.toLowerCase());
```

Comment:
Will there eventually be a `sentry.javascript.vue` value?

Reply:
Currently it uses `sentry.javascript.browser`, but angular + ember coming soon will need to be added.

Maybe if we ever decide to have a vue specific package, who knows!